code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1121,3 +1121,24 @@
 	id = "corazone"
 	description = "A medication used to treat pain, fever, and inflammation, along with heart attacks."
 	color = "#F5F5F5"
+
+/datum/reagent/medicine/vitamin_supplement
```

Comment:
It's a little weird that you're adding vitamin supplements when we already have vitamins as a reagent

Reply:
The vitamins reagent is just a "healthy" reagent; supplements make up for a lack of not having enough of it through consistent bad eating rather than not having it every few minutes.